core/src/main/scala/org/apache/spark/deploy/mesos/MesosExternalShuffleService.scala

```diff
@@ -93,7 +113,8 @@ private[mesos] class MesosExternalShuffleService(conf: SparkConf, securityManage
 
   protected override def newShuffleBlockHandler(
       conf: TransportConf): ExternalShuffleBlockHandler = {
-    new MesosExternalShuffleBlockHandler(conf)
+    val cleanerIntervalS = this.conf.getTimeAsSeconds("spark.shuffle.cleanerInterval", "30s")
```

Comment:
Is it a Spark convention to add a "S" in the end to tag it as Seconds? I'm seeing more example in the code base to have it spelled out. I'd say we should be consistent and use cleanerIntervalSeconds.
Also this is only used in the Mesos shuffle service, I think should name space the configuration and also be consistent with other interval setting names, how about "spark.mesos.shuffle.cleaner.interval"